pkg/controllers/dataexport/reconcile.go

```diff
@@ -1131,7 +1131,16 @@ func (c *Controller) stageLocalSnapshotRestore(ctx context.Context, dataExport *
 		}
 		return true, c.updateStatus(dataExport, data)
 	} else if dataExport.Status.Status == kdmpapi.DataExportStatusFailed {
-		err := c.cleanupLocalRestoredSnapshotResources(dataExport, false)
+		val, err := utils.PauseCleanupResource()
+		if err != nil {
+			// since default value is 0,do nothing.
+		} else {
+			if val != 0 {
+				logrus.Infof("Starting to wait for %v minutes before cleanup of local restored snapshot resources:%v", val, dataExport.Name)
+				time.Sleep(val)
+			}
+		}
```

Comment:
```suggestion
		val, err := utils.PauseCleanupResource()
		if err == nil && val != 0 {
			logrus.Infof("Starting to wait for %v minutes before cleanup of local restored snapshot resources:%v", val, dataExport.Name)
			time.Sleep(val)
		}
```

Since the above `err!=nil` does not do anything, its better to keep the code more concise and readable

Reply:
Thanks for the suggesstion,will check on it